src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_inpaint.py

```diff
@@ -308,6 +308,25 @@ def retrieve_timesteps(
     return timesteps, num_inference_steps
 
 
+def requires_vae_latents_normalization(vae):
```

Comment:
I think we can definitely do this in place without having to delegate it to a method. As that way the readability of the code stays linear and the reader doesn't have to refer to another method to see what's going on.